src/_content/posts/tutorials.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "DjangoCon US 2024 Tutorials"
+layout: post
+
+author: Lidya K
+category: General
+published_datetime: 2024-08-18 07:56:00-04:00
+---
+## We’re thrilled to announce the highly anticipated DjangoCon US 2024 Tutorials! 🐍
+
+🗓️ Date: Sunday, September 22
+
+📍 Locations: Durham Convention CenterDurham, NC
+
+Gear up to boost your Django skills and elevate your web development expertise with a wide range of in-depth tutorials led by industry professionals. Whether you're an experienced Django developer or just beginning your journey, DjangoCon US 2024 offers something for everyone.
+Here’s a sneak peek of what’s in store:
+
+### 8:00 am - 9:00 am
+
+Grand Ballroom I/II Continental Breakfast
+
+### 8:00 am to 6:00 pm
+
+In front of Junior Ballroom Registration
+
+### 9:30 am - 1:00 pm
+
+#### Tutorial Track A
+
+Led by Kuldeep Pisda Description: Efficient Django: Mastering Asynchronous Tasks with Celery, RabbitMQ, and Redis
```

Comment:
Maybe a better format might be `[tutorial title] led by [presenter name]`. Then link the tutorials to the actual talks.

Reply:
yeah, this feels awkward as it is